plugins/module_utils/network/sonic/config/logging/logging.py

```diff
@@ -325,3 +440,17 @@ def get_delete_all_servers_requests(self):
         requests.append(request)
 
         return requests
+
+    def send_requests(self, requests):
```

Comment:
I think you should move this function to utils to avoid repetition for when others implement overridden and replaced